Mustache/gulpfile.js

```diff
@@ -0,0 +1,74 @@
+// Import Gulp and Plumber
+const gulp = require('gulp');
+
+// SASS compilation task
+const plumber = require('gulp-plumber');
+const sass = require('gulp-sass');
+const concat = require('gulp-concat');
+const cssnano = require('gulp-cssnano');
+
+gulp.task('sass', function () {
+	gulp.src('./src/css/*.scss')
+		.pipe(plumber())
+		.pipe(sass())
+		.pipe(cssnano())
+		.pipe(concat('main.css'))
+		.pipe(gulp.dest('./dist'));
+});
+
+// BrowserSync serve task
+const browser = require('browser-sync');
+const reload = browser.reload;
+gulp.task('serve', function () {
+	browser({
+		port: 4500,
+		open: false,
+		ghostMode: false,
+		server: {
+			baseDir: './dist'
+		}
+	});
+});
+
+// Watch task
+gulp.task('watch', function () {
+	gulp.watch("./src/css/**", ['sass']);
+	gulp.watch("./src/html/**", ['html']);
+	gulp.watch("./src/js/**", ['js']);
+});
+
+// HTML generation task
+// var fs = require("fs");
```

Comment:
Don't leave commented lines. It may confuse the development of future changes (like "is this line needed for something?", "Should I uncomment this line to build the project and then comment it?", etc). Confusing for others, and also for you.

Try not to leave them, but If you really need to, think about leaving a small comment of why that line should be there.